### PR TITLE
virtio_fs: stop/start virtiofs service for windows guest

### PR DESCRIPTION
**Pull request by xiagao**
Add a new case to repeate stop/start virtiofs service for windows guest.
ID: 2092226
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
--testcase=virtio_fs_share_data..viofs_service_stop_start --guestname=Win2019
 (1/1) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto.viofs_service_stop_start.q35: STARTED
 (1/1) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto.viofs_service_stop_start.q35: PASS (171.09 s)

@YongxueHong @leidwang  Could you help to review?

File: qemu/tests/virtio_fs_share_data.py [hunk 1]
```diff
@@ -488,9 +519,26 @@ def is_autoit_finished(session, process_name):
...
                         test.fail("Git init failed with %s" % output)
+
+                if params.get("stop_start_repeats") and os_type == "windows":
+                    repeats = int(params.get("stop_start_repeats", 1))
+                    for i in range(repeats):
+                        error_context.context("Repeat stop/start VirtioFsSvc:"
+                                              " %s/%s" % (i + 1, repeats),
+                                              test.log.info)
+                        viofs_svc_stop_start("stop", viofs_sc_stop_cmd)
+                        time.sleep(1)
```
[the remark this answers is not in the document]
Comment (leidwang) [reply qemu/tests/virtio_fs_share_data.py:532]: Hi @YongxueHong Could you give some suggestions about that?Thanks!

[the remark this answers is not in the document]
Comment (YongxueHong) [reply qemu/tests/virtio_fs_share_data.py:532]: > Hi @YongxueHong Could you give some suggestions about that?Thanks!

Yes, I also think it is better to add a step to check whether the status of service is correct.

[the remark this answers is not in the document]
Comment (xiagao) [reply qemu/tests/virtio_fs_share_data.py]: I think there is no need to check the status again.
For windows os, sc start servicename or stop it, the result will contain the status.
 sc start VirtioFsSvc
2022-07-21 04:45:32: SERVICE_NAME: VirtioFsSvc
2022-07-21 04:45:32:         TYPE               : 10  WIN32_OWN_PROCESS
2022-07-21 04:45:32:         STATE              : 1  STOPPED
2022-07-21 04:45:32:         WIN32_EXIT_CODE    : 0  (0x0)
2022-07-21 04:45:32:         SERVICE_EXIT_CODE  : 0  (0x0)
2022-07-21 04:45:32:         CHECKPOINT         : 0x0
2022-07-21 04:45:32:         WAIT_HINT          : 0x0

Sending command: sc stop VirtioFsSvc
2022-07-21 04:45:32: SERVICE_NAME: VirtioFsSvc
2022-07-21 04:45:32:         TYPE               : 10  WIN32_OWN_PROCESS
2022-07-21 04:45:32:         STATE              : 4  RUNNING
2022-07-21 04:45:32:                                 (NOT_STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)
2022-07-21 04:45:32:         WIN32_EXIT_CODE    : 0  (0x0)
2022-07-21 04:45:32:         SERVICE_EXIT_CODE  : 0  (0x0)
2022-07-21 04:45:32:         CHECKPOINT         : 0x0
2022-07-21 04:45:32:         WAIT_HINT          : 0x0
2022-07-21 04:45:32:         PID                : 1944
2022-07-21 04:45:32:         FLAGS              :

And @leidwang ,about the sleep time, it works without sleep 1 second.

[the remark this answers is not in the document]
Comment (leidwang) [reply qemu/tests/virtio_fs_share_data.py]: I noticed that you add a checkpoint(`if status != 0 or expect_status not in output:`) in the function(`viofs_svc_stop_start`),and removed the `time.sleep`,it is okay for me.

**Review by leidwang — APPROVED** [review]

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Pull request merged**